rest/server_context.go

```diff
@@ -408,6 +408,7 @@ type PostUpgradeResult map[string]PostUpgradeDatabaseResult
 type PostUpgradeDatabaseResult struct {
 	RemovedDDocs   []string `json:"removed_design_docs"`
 	RemovedIndexes []string `json:"removed_indexes"`
+	err            error    // this is not exported, to be used for additional information
```

Comment:
I don't see this being used anywhere - is it still required?  I don't think the audit event needs to include whether removal of design docs or indexes occurred.

Reply:
I made this event more complicated than it needed to be in a previous review, I moved post upgrade from being db specific.